codeship-services.yml

```diff
@@ -1,8 +1,8 @@
 php:
-  image: silintl/php7
+  image: silintl/php7-apache:7.4.19
```

Comment:
I've ended up using `silintl/php7:7.4` now, since it seems to perform better and be more stable than `silintl/php7-apache`. Not critical, but probably worth switching at some point.